### PR TITLE
Add issue template, PR template

### PR DESCRIPTION
**Pull request by ilaflott**
## Describe your changes
Adding two templates for issues, a bug report and feature request, both stock from github. Adding also a PR template, this PR is using the proposed format.

## Issue ticket number and link (if applicable)
https://github.com/NOAA-GFDL/fre-cli/issues/119

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I wrote a test, if applicable
- [x] I ran pylint and tried to address some of it's complaints

### CODE REVIEW

**Comment by ilaflott** [comment]
Would love feedback on these templates!

**Review by ceblanton — APPROVED** [review]
Thanks! Completely agree.

File: .past_files/helpful_docs/revisions.txt [hunk 1]
```diff
@@ -1,329 +0,0 @@
-commit d971c4d4a80dd1a64480dd22b931f4f61000594c
```
Comment (singhd789) [inline .past_files/helpful_docs/revisions.txt:1]: I didn't notice this file! Was this just the git history of Bennett changes?

**Review by singhd789 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (singhd789) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: Would bug_report and freture_request go in the same issue that would be opened?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Describe your changes
+
+## Issue ticket number and link (if applicable)
+
+## Checklist before requesting a review
```
Comment (singhd789) [inline .github/pull_request_template.md:5]: Should we add `documentation written`  (if necessary/applicable) on the checklist? If a test is written or code updated, I think some documentation should be updated with it.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (ilaflott) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: pick-and-choose! If one really feels the need to use both, I would think that's two separate issues to be opened. Unless they're really that inseparable, somehow.

File: .github/pull_request_template.md (hunk 3 above)
Comment (ilaflott) [reply .github/pull_request_template.md:5]: good catch, addressing.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (singhd789) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: Gotcha!

**Comment by ilaflott** [comment]
OK happy with the cleanup i could pull off here. merging.

**Pull request merged**